src/app/containers/Bulletin/index.stories.jsx

```diff
@@ -33,33 +35,50 @@ const audioFixture = bulletinFixture('RadioBulletin');
 const liveTvFixture = liveBulletinFixture('TVBulletin');
 const audioLiveFixture = liveBulletinFixture('RadioBulletin');
 
-const getBulletinPromo = platform => item => (
-  <ServiceContextProvider service="news">
+const getBulletinPromo = (platform, service, item) => (
+  <ServiceContextProvider service={service}>
     <RequestContextProvider
       bbcOrigin="https://www.test.bbc.co.uk"
       isAmp={platform === 'amp'}
       pageType="frontPage"
-      service="news"
+      service={service}
```

Comment:
`pathname` needs to be defined, as the lack of it is causing this console warning `Warning: Failed prop type: The prop `pathname` is marked as required in 'RequestContextProvider', but its value is 'undefined'.`

Reply:
Sorry, hadn't refreshed my page to see the previous comment